File: ejercicio 3.cpp
#include <stdio.h>
#include <stdlib.h>

int main() {
    char sucursales[25][20];
    float ventas[25], suma_ventas = 0.0, promedio;

    for (int i = 0; i < 25; i++) {
        snprintf(sucursales[i], sizeof(sucursales[i]), "Sucursal %d", i + 1);
    }

    for (int i = 0; i < 25; i++) {
        printf("Ingrese las ventas de la %s: ", sucursales[i]);
        scanf("%f", &ventas[i]);
        suma_ventas += ventas[i];
    }

    promedio = suma_ventas / 25;

    printf("Promedio de ventas: %.2f\n", promedio);

    printf("Sucursales con ventas mayores al promedio:\n");
    for (int i = 0; i < 25; i++) {
        if (ventas[i] > promedio) {
            printf("%s - Ventas: %.2f\n", sucursales[i], ventas[i]);
        }
    }

    return 0;
}

File: ejercicio 3.py
import random
sucursales = [f"Sucursal {i+1}" for i in range(25)]
ventas = [random.randint(1000, 5000) for _ in range(25)]

promedio_ventas = sum(ventas) / len(ventas)
sucursales_mayores_promedio = []

for i in range(len(ventas)):
    if ventas[i] > promedio_ventas:
        sucursales_mayores_promedio.append(sucursales[i])

print(f"Promedio de ventas: {promedio_ventas}")
print("Sucursales con ventas mayores al promedio:")
for sucursal in sucursales_mayores_promedio:
    print(sucursal)
